demos/python_demos/multi_camera_multi_person_tracking/mc_tracker/sct.py

```diff
@@ -0,0 +1,405 @@
+            self.tracks[i]['avg_feature'] = (none_to_zero(self.tracks[i]['avg_feature'])*len(self.tracks[i]['features']) + \
+                                             none_to_zero(self.tracks[idx]['avg_feature'])*len(self.tracks[idx]['features']))
+            self.tracks[i]['f_cluster'] = self._merge_clustered_features(
+                                            self.tracks[i]['f_cluster'],
+                                            self.tracks[idx]['f_cluster'],
+                                            self.tracks[i]['avg_feature'],
+                                            self.tracks[idx]['avg_feature'])
+            self.tracks[i]['timestamps'] += self.tracks[idx]['timestamps']
+            self.tracks[i]['boxes'] += self.tracks[idx]['boxes']
+            self.tracks[i]['features'] += self.tracks[idx]['features']
+            self.tracks[i]['avg_feature'] /= len(self.tracks[i]['features'])
+            self.tracks[idx] = None
+        else:
+            assert self.tracks[idx]['timestamps'][-1] <= self.tracks[i]['timestamps'][0]
+            self.tracks[idx]['avg_feature'] = (none_to_zero(self.tracks[i]['avg_feature'])*len(self.tracks[i]['features']) + \
+                                               none_to_zero(self.tracks[idx]['avg_feature'])*len(self.tracks[idx]['features']))
+            self.tracks[idx]['f_cluster'] = self._merge_clustered_features(
+                                                self.tracks[i]['f_cluster'],
+                                                self.tracks[idx]['f_cluster'],
+                                                self.tracks[i]['avg_feature'],
+                                                self.tracks[idx]['avg_feature'])
+            self.tracks[idx]['timestamps'] += self.tracks[i]['timestamps']
+            self.tracks[idx]['boxes'] += self.tracks[i]['boxes']
+            self.tracks[idx]['features'] += self.tracks[i]['features']
+            self.tracks[idx]['avg_feature'] /= len(self.tracks[idx]['features'])
+            self.tracks[i] = None
+
+    def _create_new_tracks(self, detections, features, assignment):
+        assert len(detections) == len(features)
+        for i, j in enumerate(assignment):
+            if j is None:
+                self.tracks.append(self._create_tracklet_descr(self.time,
+                                                               detections[i],
+                                                               self.global_id_getter(),
+                                                               features[i]))
+
+    def _create_tracklet_descr(self, timestamp, rect, id, feature):
+        return {'id': id,
+                'cam_id': self.id,
+                'boxes': [rect],
+                'timestamps': [timestamp],
+                'features': [feature],
+                'avg_feature': feature.copy() if feature is not None else None,
+                'f_cluster': ClusterFeature(self.n_clusters, feature)}
```

Comment:
I suggest using `namedtuple` instead of dict.

Reply:
I need to have a mutable collection here, so I'd rather try the `recordclass`.

Reply:
I think it's OK to use dict than.